app/assets/javascripts/discourse/app/widgets/actions-summary.js

```diff
@@ -1,40 +1,24 @@
 import { hbs } from "ember-cli-htmlbars";
-import RenderGlimmer, {
-  registerWidgetShim,
-} from "discourse/widgets/render-glimmer";
-import { createWidget } from "discourse/widgets/widget";
+import { registerWidgetShim } from "discourse/widgets/render-glimmer";
 
-createWidget("small-user-list", {
-  tagName: "div.clearfix.small-user-list",
-
-  buildClasses(atts) {
-    return atts.listClassName;
-  },
-
-  buildAttributes(attrs) {
-    const attributes = { role: "list" };
-    if (attrs.ariaLabel) {
-      attributes["aria-label"] = attrs.ariaLabel;
-    }
-    return attributes;
-  },
-
-  html(attrs) {
-    return [
-      new RenderGlimmer(
-        this,
-        "span.small-user-list-content",
-        hbs`<SmallUserList @data={{@data.attrs}}/>`,
-        {
-          attrs,
-        }
-      ),
-    ];
-  },
-});
+// TODO check the rendering in the two voting plugins
+// This shim is nesting everything into a DIV and changing the HTML but only thw two voting plugins
+// are using this widget outside of core.
```

Comment:
Should we check the voting plugins before merging?

Reply:
I checked how both plugins are rendering the widget and it's fine.